packages/create-next-app/index.ts

```diff
@@ -254,40 +254,33 @@ async function run(): Promise<void> {
       preferences[field] ?? defaults[field]
 
     if (!program.typescript && !program.javascript) {
-      if (ciInfo.isCI) {
```

Comment:
Do we still need this check so we don't attempt prompting in CI like the other checks?

Reply:
Good point, I pushed a change [993fb10](https://github.com/vercel/next.js/pull/53686/commits/993fb10f366876ae15a706c8d0b558d0a6597573)